site/develop/committers.md

```diff
@@ -51,6 +51,7 @@ Vaibhav Gumashta        | vgumashta    | Committer
 Wei Zheng               | weiz         | Committer
 William Hyun            | william      | PMC Chair
 Xiening Dai             | xndai        | Committer
+Xin Zhang               | xinzhang     | Committer
```

Comment:
Thank you. Please sort by Apache ID as mentioned here, @coderex2522 .
https://github.com/apache/orc/blob/e674d8a61d77d0e477c5251f5091b07e1c3c1d3e/site/develop/committers.md#L9